mindsdb/utilities/api_status.py

```diff
@@ -0,0 +1,49 @@
+import os
+import tempfile
+import json
+import time
+
+
+_api_status_file = None
+
+
+def get_api_status_file():
+    global _api_status_file
+    if _api_status_file is None:
+        # Use a temporary file that can be shared across processes.
+        temp_dir = tempfile.gettempdir()
+        _api_status_file = os.path.join(temp_dir, "mindsdb_api_status.json")
```

Comment:
mindsdb server creates `mindsdb` folder in temp directore. this file can be located there